openslides/core/management/commands/collectstatic.py

```diff
@@ -0,0 +1,42 @@
+import os
+from typing import Any, Dict
+
+from django.conf import settings
+from django.contrib.staticfiles.management.commands.collectstatic import Command as CollectStatic
+from django.core.management.base import CommandError
+from django.db.utils import OperationalError
+
+from ...views import WebclientJavaScriptView
+
+
+class Command(CollectStatic):
+    """
+    Custom collectstatic command.
+    """
+    realms = ['site', 'projector']
+    js_filename = 'webclient-{}.js'
+
+    def handle(self, **options: Any) -> str:
+        try:
+            self.view = WebclientJavaScriptView()
+        except OperationalError:
+            raise CommandError('You have to run OpenSlides first to create a ' +
+            'database before collecting staticfiles.')
+        return super().handle(**options)
+
+    def collect(self) -> Dict[str, Any]:
+        destination_dir = os.path.join(settings.OS_STATICFILES_DIR, 'js')
```

Comment:
So here we take the first entry of `settings.STATICFILES_DIR` (and of course skip the code, if it does not exist).

Reply:
@FinnStutzenstein ?